components/nft/MintedNFTs.tsx

```diff
@@ -47,7 +47,7 @@ const MintedNFTs: FC<{
                                 isLoading: isUserMinting,
                                 as: 'button',
                                 variant: 'primary',
-                                onClick: () => mintAccount().then(fetchNfts),
+                                onClick: () => mintAccount(),
```

Comment:
I dont see that `mintAccount` updates the NFTs after a mint success, if there is then perfect, otherwise we need to keep the previous logic `onClick: () => mintAccount().then(fetchNfts),`

Reply:
When an nft is minted it takes a while to propagate in the blockchain, so if you minted the nft and fetchNfts right after, the minted nft will not be seen. 
As this functionality takes a long time, I decided to remove this fetch. A text message could be put to notify the user that they can refresh to see the new nft, the same like when creating a candy machine, what do you think?

Reply:
sounds good 👍🏼 lets leave it as is and if we feel it needs an extra message we can add it later